Content.Server/Administration/Commands/PlayGlobalSound.cs

```diff
@@ -0,0 +1,57 @@
+using Content.Shared.Administration;
+using Robust.Server.Player;
+using Robust.Shared.Audio;
+using Robust.Shared.Console;
+using Robust.Shared.Player;
+
+namespace Content.Server.Administration.Commands;
+
+/// <summary>
+///     Command that allows admins to play global sounds.
+/// </summary>
+[AdminCommand(AdminFlags.Fun)]
+public sealed class PlayGlobalSound : IConsoleCommand
+{
+    public string Command => "playglobalsound";
+    public string Description => Loc.GetString("play-global-sound-command-description");
+    public string Help => Loc.GetString("play-global-sound-command-help");
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        Filter filter;
+
+        switch (args.Length)
+        {
+            // No arguments, show command help.
+            case 0:
+                shell.WriteLine(Help);
+                return;
+
+            // No users, play sound for everyone.
+            case 1:
+                filter = Filter.Broadcast();
+                break;
+
+            // One or more users specified.
+            default:
+                var playerManager = IoCManager.Resolve<IPlayerManager>();
```

Comment:
dunno how i missed this. Use a dependency.

Reply:
Addressed in 8d2c43c4e0445ddbd64c96fafcbc8724580c4fca